homework/index.js

```diff
@@ -29,22 +29,47 @@
     return elem;
   }
 
+  function createLiElement(ul, element1, element2, options1, options2) {
+    const li = createAndAppend('li', ul, {class: 'list'});
+    return createAndAppend(element1, li, options1),
+      createAndAppend(element2, li, options2);
+  }
+
+  function dateChange(date){
```

Comment:
You are not "changing" the date, you are just formatting the date in a user presentable format. So a name such as `formatDate` would be a more accurate description.